src/main.rs

```diff
@@ -201,10 +203,7 @@ fn main() -> Result<()> {
 
         return Ok(());
     } else if let Some(ide_complete) = parsed_nu_cli_args.ide_complete {
-        let cwd = std::env::current_dir().expect("Could not get current working directory.");
-        engine_state.add_env_var("PWD".into(), Value::test_string(cwd.to_string_lossy()));
```

Comment:
I think removing these lines here could be problematic since `PWD` isn't being set anymore.